lib/ohai/plugins/linux/platform.rb

```diff
@@ -72,6 +72,54 @@ def os_release_file_is_cisco?
     File.exist?("/etc/os-release") && os_release_info["CISCO_RELEASE_INFO"]
   end
 
+  #
+  # Determines the platform version for Debian based systems
+  #
+  # @returns [String] version of the platform
+  #
+  def debian_platform_version
+    if platform == "cumulus"
+      if File.exist?("/etc/cumulus/etc.replace/os-release")
+        release_contents = File.read("/etc/cumulus/etc.replace/os-release")
+        release_contents.match(/VERSION_ID=(.*)/)[1] rescue nil
```

Comment:
Probably. I keep going back and forth with how much Ohai should warn when things don't go right. Recently leaning more to towards Ohai being verbose when things fail so that people see that things are failing and file issues. We have a lot of hidden failures right now, which doesn't help anyone.